front-end/src/pages/FollowListPage.tsx

```diff
@@ -0,0 +1,93 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import { Row, Col, Card, CardBody, CardTitle, Container, Button, CardText, Nav, NavItem, NavLink, TabContent, TabPane } from 'reactstrap';
+import AuthorListItem from '../components/AuthorListItem';
+import { Author } from '../types/Author';
+import { UserLogin } from '../types/UserLogin';
+
+interface Props extends RouteComponentProps<MatchParams>{
+  loggedInUser: UserLogin | undefined,
+  activeTab: string,
+}
+
+interface MatchParams {
+  authorId: string;
+}
+/**
+ * Render list of search results when searching for an author by display name
+ * @param props 
+ */
+export default function FollowListPage(props: Props) {
+  const [followers, setFollowers] = useState<Author[] | undefined>(undefined);
+  const [following, setFollowing] = useState<Author[] | undefined>(undefined);
+
+  // get all followers and following
+  useEffect(() => {
+    axios.get(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser?.authorId + "/followers/").then(res => {
+      console.log(res.data.items)
+      const followersList: Author[] = res.data.items;
+      setFollowers(followersList);
+    });
+
+    axios.get(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser?.authorId + "/following/").then(res => {
+      console.log(res.data)
+      const followingList: Author[] = res.data.items;
+      setFollowing(followingList);
+    });
+
+  }, []);
+
+  const [activeTab, setActiveTab] = useState(props.activeTab);
+
+  const toggle = (tab: string) => {
+    if (activeTab !== tab) setActiveTab(tab);
+    props.history.push("/author/" + props.loggedInUser?.authorId + "/" + tab);
+  }
+
+  return (
+    <Container fluid>
+
+      <Row className="justify-content-md-center">
+        <Nav tabs>
+          <NavItem>
+            <NavLink onClick={() => { toggle('followers') }} className="mb-2 text-muted">
+              <h2>Followers</h2>
+            </NavLink>
+          </NavItem>
+          <NavItem>
+            <NavLink onClick={() => { toggle('following') }} className="mb-2 text-muted">
+              <h2>Following</h2>
+            </NavLink>
+          </NavItem>
+        </Nav>
+      </Row>
+      <Row className="justify-content-md-center">
+        <Col sm="12" md={{ size: 6 }}>
+          <TabContent activeTab={activeTab}>
+            <TabPane tabId="followers">
+              <Row>
```

Comment:
The positioning might be a bit off if we don't wrap `<Row>` els with `<Containter>`

Reply:
The whole page is already in a container

Reply:
Correct, though currently, iirc it's
```jsx
<Container>
    <Row>
        <Col>
           <Row>
                ...
           </Row>
        </Col>
    </Row>
</Container>
```
we should have it like this, or else the CSS can act up, since we usually don't put `<Row>` elements directly in `<Col>` elements
```jsx
<Container>
    <Row>
        <Col>
           <Container>
               <Row>
                ...
               </Row>
           </Container>
        </Col>
    </Row>
</Container>
```

Reply:
fixed